scripts/updateMarketConfigUtils.ts

```diff
@@ -1,33 +1,39 @@
 import hre, { network } from "hardhat";
 
-import { validateMarketConfigs } from "./validateMarketConfigsUtils";
-import { encodeData } from "../utils/hash";
-import { bigNumberify } from "../utils/math";
-import { getMarketKey, getMarketTokenAddresses, getOnchainMarkets } from "../utils/market";
-import { getFullKey, appendUintConfigIfDifferent } from "../utils/config";
+import { MockRiskOracle } from "../typechain-types";
 import { handleInBatches } from "../utils/batch";
+import { appendUintConfigIfDifferent, getFullKey } from "../utils/config";
+import { encodeData } from "../utils/hash";
 import * as keys from "../utils/keys";
+import { getMarketKey, getMarketTokenAddresses, getOnchainMarkets } from "../utils/market";
+import { bigNumberify } from "../utils/math";
+import { validateMarketConfigs } from "./validateMarketConfigsUtils";
 
 const RISK_ORACLE_MANAGED_BASE_KEYS = [keys.MAX_OPEN_INTEREST];
 
 const processMarkets = async ({
   markets,
   onchainMarketsByTokens,
+  supportedByRiskOracleMarkets,
   tokens,
   generalConfig,
   handleConfig: handleConfigArg,
 }) => {
-  const shouldHandleBaseKey = (baseKey: string) => {
-    if (hre.network.name !== "arbitrum") {
+  const shouldHandleBaseKey = (baseKey: string, isSupportedByRiskOracle: boolean) => {
+    if (hre.network.name !== "arbitrum" && hre.network.name !== "avalancheFuji") {
```

Comment:
may be clearer to add a const SUPPORTED_MARKETS above instead, similar to RISK_ORACLE_MANAGED_BASE_KEYS